apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationStatement.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react'
+import django from 'django'
+
+export const ModerationStatement = (props) => {
+  const [statement, setStatement] = useState(props.initialStatement)
+  const placeholderText = django.pgettext('kosmo', 'Write statement')
+  const charCountText = django.pgettext('kosmo', ' characters')
+
+  const handleSubmit = (e) => {
+    e.preventDefault()
+    const statement = this.state.statement.trim()
```

Comment:
I believe `this` won't work here anymore.
`this.state.statement` is now simply `statement` i guess.

Reply:
same for `this.props`, is now `props` in functional components

Reply:
statement caused errors, will wait for api before i mess around here as I just straight up copied it from old component